OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -1724,6 +1724,8 @@ static class DMOpcodeHandlers {
                     return first.GetValueAsFloat() > second.GetValueAsFloat();
                 case DreamValue.DreamValueType.Float when second.Value == null:
                     return first.GetValueAsFloat() > 0;
+                case DreamValue.DreamValueType.String when second.Type == DreamValue.DreamValueType.String:
+                    return string.Compare(first.GetValueAsString(), second.GetValueAsString()) > 0;
```

Comment:
The `StringComparison.Ordinal` comparison type needs to be used to sort by binary value.
```suggestion
                    return string.Compare(first.GetValueAsString(), second.GetValueAsString(), StringComparison.Ordinal) > 0;
```